internal/btf/btf_types.go

```diff
@@ -61,6 +62,45 @@ type btfType struct {
 	SizeType uint32
 }
 
+func (k btfKind) String() string {
+	switch k {
+	case kindUnknown:
+		return "Unknown"
+	case kindInt:
+		return "Int"
+	case kindPointer:
+		return "Pointer"
+	case kindArray:
+		return "Array"
+	case kindStruct:
+		return "Struct"
+	case kindUnion:
+		return "Union"
+	case kindEnum:
+		return "Enum"
+	case kindForward:
+		return "Forward"
+	case kindTypedef:
+		return "Typedef"
+	case kindVolatile:
+		return "Volatile"
+	case kindConst:
+		return "Const"
+	case kindRestrict:
+		return "Restrict"
+	case kindFunc:
+		return "Func"
+	case kindFuncProto:
+		return "FuncProto"
+	case kindVar:
+		return "Var"
+	case kindDatasec:
+		return "Datasec"
```

Comment:
`linux/btf.h` calls this just `Section`
```suggestion
		return "Section"
```